torch/distributed/distributed_c10d.py

```diff
@@ -1190,10 +1190,7 @@ def get_backend_config(group: Optional[ProcessGroup] = None) -> str:
         The backend configuration of the given process group as a lower case string.
 
     """
-    if group is None:
-        pg = _get_default_group()
-    else:
-        pg = group
+    pg = group or _get_default_group()
```

Comment:
To retain the exact same semantic but still make it a bit shorter, can be `pg = _get_default_group() if group is None else group`